tests/baselines/reference/tsserver/projects/no-tsconfig-script-block-diagnostic-errors.js

```diff
@@ -595,7 +603,8 @@ Info seq  [hh:mm:ss:mss] Creating ConfiguredProject: /user/username/projects/pro
 Info seq  [hh:mm:ss:mss] FileWatcher:: Added:: WatchInfo: /user/username/projects/project/tsconfig.json 2000 undefined Project: /user/username/projects/project/tsconfig.json WatchType: Config file
 Info seq  [hh:mm:ss:mss] Config: /user/username/projects/project/tsconfig.json : {
  "rootNames": [
-  "/user/username/projects/project/f1.ts"
+  "/user/username/projects/project/f1.ts",
+  "/user/username/projects/project/f2.html"
```

Comment:
This seems to be legit, based on the issue. Extensions marked as anything but Deferred where not considered as part of the root of project.